src/llvm/message.rs

```diff
@@ -8,7 +8,7 @@ use llvm_sys::core::LLVMDisposeMessage;
 /// Does not own the ptr, so we have to call `LLVMDisposeMessage` to free message memory.
 #[repr(C)]
 pub struct Message {
-    pub ptr: *mut i8,
+    pub ptr: *mut ::libc::c_char,
```

Comment:
add a `use libc::c_char;` declaration at the top and use `c_char` here and in other places please